mail/rspamd/src/opnsense/service/templates/OPNsense/Rspamd/redis.conf

```diff
@@ -16,7 +16,7 @@
 {% if helpers.exists('OPNsense.Rspamd.general.enable_redis_plugin') and OPNsense.Rspamd.general.enable_redis_plugin == '1' %}
 {%   if helpers.exists('OPNsense.redis.general.enabled') and OPNsense.redis.general.enabled == '1' %}
 
-servers = "[::1]{% if helpers.exists('OPNsense.redis.general.port') and OPNsense.redis.general.port != '' %}:{{ OPNsense.redis.general.port }}{% endif %}";
+servers = "/var/run/redis/redis.sock";
```

Comment:
I guess it makes more sense, if it would be possible to support both.
The redis plugin is designed to act as infrastructure for other plugins. Maybe they do not support unix domain sockets.